apps/passwd.c

```diff
@@ -556,7 +556,7 @@ static char *shacrypt(const char *passwd, const char *magic, const char *salt)
     OPENSSL_strlcat(out_buf, "$", sizeof out_buf);
     if (rounds_custom) {
         char tmp_buf[7 + 9 + 1]; /* "rounds=999999999" */
-        sprintf(tmp_buf, "rounds=%lu", rounds);
+        sprintf(tmp_buf, "rounds=%zu", rounds);
```

Comment:
Not all platforms have support for `zu`. Please use the `OSSLzu` macro instead defined in e_os2.h